source/vibe/textfilter/urlencode.d

```diff
@@ -62,7 +64,7 @@ private bool isCorrectHexNum(string str)
 
 /** Checks whether a given string has valid URL encoding.
 */
-bool isURLEncoded(string str, string reserved_chars = null)
+auto isURLEncoded(const(char)[] str, const(char)[] reserved_chars = null)
```

Comment:
But in this case I'd rather put the attributes manually, so that both, the return type and the attributes appear in the documentation.